src/main/java/com/prgms/voucher/voucherproject/domain/FixedAmountVoucher.java

```diff
@@ -0,0 +1,30 @@
+package com.prgms.voucher.voucherproject.domain;
+
+import java.util.UUID;
+
+public class FixedAmountVoucher implements Voucher{
```

Comment:
띄어쓰기가 빠져있어서 코드 제안 드렸어요.
```suggestion
public class FixedAmountVoucher implements Voucher {
```